google-cloud-storage/src/main/java/org/jclouds/googlecloudstorage/features/ObjectApi.java

```diff
@@ -199,7 +198,7 @@ GoogleCloudStorageObject simpleUpload(@PathParam("bucket") String bucketName, @H
    @Named("Object:delete")
    @DELETE
    @Path("storage/v1/b/{bucket}/o/{object}")
-   @Fallback(TrueOnNotFoundOr404.class)
+   @Fallback(FalseOnNotFoundOr404.class)
```

Comment:
Is the current ``master`` GCS test suite supposed to pass? I'm having a few errors, in particular in ``ObjectApiLiveTest`` (the class I'm working on).